index.d.ts

```diff
@@ -185,47 +186,51 @@ declare global {
 
   export type GPUTextureAspect = "all" | "stencil-only" | "depth-only";
 
-  export type GPUBufferUsageFlags = number;
-  export enum GPUBufferUsage {
-    NONE = 0x0000,
-    MAP_READ = 0x0001,
-    MAP_WRITE = 0x0002,
-    COPY_SRC = 0x0004,
-    COPY_DST = 0x0008,
-    INDEX = 0x0010,
-    VERTEX = 0x0020,
-    UNIFORM = 0x0040,
-    STORAGE = 0x0080,
-    INDIRECT = 0x0100
+  interface GPUBufferUsageInterface {
+    MAP_READ:  0x0001;
+    MAP_WRITE: 0x0002;
+    COPY_SRC:  0x0004;
+    COPY_DST:  0x0008;
+    INDEX:     0x0010;
+    VERTEX:    0x0020;
+    UNIFORM:   0x0040;
+    STORAGE:   0x0080;
+    INDIRECT:  0x0100;
   }
+  export type GPUBufferUsageFlags = number;
+  export type GPUBufferUsage = GPUBufferUsageInterface[keyof GPUBufferUsageInterface];
```

Comment:
open question: is this a useful type definition? or should users just use GPUBufferUsageFlags? (same elsewhere)

Reply:
Talked with Austin and removed.